### ISSUE TITLE
Server Connection Error Popup Box

### ISSUE THREAD

**Issue by jpwalters** [jpwalters/OpenVBX-Plugin-SMS-Conversation#13]
There's an occasionally error message that will popup randomly.  I guess that likely an ajax call is timing out and we just need to trap and supress the error.  Unfortunately I can't re-create the problem on demand so its a little hard to fix.

The image below is a similar but not exactly the error message that's seen.
**Error Message:**
![image](https://cloud.githubusercontent.com/assets/4819310/14096990/f5099bd0-f53a-11e5-9bdb-f3d6ff1febcd.png)

**Comment by gegere** [comment]
This problem can be easier reproduce with a change to the connection URL within the plugin using a browser inspector tool. One possible idea is to have an on focus call or a push from the script when a new message is received and not have the script check repeatedly.

**Comment by bkonia** [comment]
I'm seeing this as well. It seems to happen after my internet connection is lost. If it's unable to make a connection, it should simply keep retrying, rather than displaying this error message. When the internet connection comes back online, it should gracefully reconnect, without bothering the user about it. You could make it display a little icon to indicate the online/offline status.

**Comment by jpwalters** [comment]
Yes, that's also when I've noticed this error the most.  I'll probably call that the issue and just make sure that when the browser cannot connect to the server that we suppress.the error.

**Comment by gegere** [comment]
@bkonia +1

**Comment by jpwalters** [comment]
The error is begin generated by a global js script in OpenBX.  Below is the link to the code and function.  Any thoughts on how I can override this to keep it from popping up within this plugin?

https://github.com/twilio/OpenVBX/blob/master/assets/j/global.js#L248
